docs/MO_DG/prepare_model/customize_model_optimizer/Customize_Model_Optimizer.md

```diff
@@ -10,6 +10,11 @@
 
 @endsphinxdirective
 
+
+This article describes Model Optimizer internals. Altering them may result in application instability, and in case of future changes to the API, lack of backward compatibility.
+
+> **NOTE**: If you want to add support for ONNX or PaddlePaddle operations, or you are not familiar with other extension alternatives in OpenVINO, read [this guide](../../../Extensibility_UG/Intro.md) instead.
```

Comment:
I think we can add TensorFlow in this list as well. @rkazants, can we?

Reply:
@slyalin, yes, it should work.